frontend/common/src/components/Pagination/Pagination.tsx

```diff
@@ -37,7 +54,7 @@ const Pagination: React.FunctionComponent<PaginationProps> = ({
   });
 
   // If there are less than 2 times in pagination range we shall not render the component
-  if (currentPage === 0 || paginationRange.length < 2) {
+  if (currentPage === 0 || totalCount < 2) {
     return null;
   }
```

Comment:
Does this work at the moment? Pagination thing appears with less than two items, or have I misunderstood what this is supposed to do.

![image](https://user-images.githubusercontent.com/40485260/160723516-86b98185-1e7d-4ea0-bc25-65787006a1e6.png)

Reply:
Hmm, so I was trying to fix the problem of the pagination component suddenly disappearing when a user changes the number of how many items to show (the select box at the end). I'll ask what the designers want to do about that..